Assets/Scripts/StandardSamples/UI/Login/UILoginMenu.cs

```diff
@@ -687,11 +687,15 @@ private void ConfigureUIForConnectLogin()
             {
 
                 //case ExternalCredentialType.GogSessionTicket:
-                //case ExternalCredentialType.GoogleIdToken:
                 //case ExternalCredentialType.ItchioJwt:
                 //case ExternalCredentialType.ItchioKey:
                 //case ExternalCredentialType.AmazonAccessToken:
-
+#if !UNITY_ANDROID || UNITY_EDITOR
```

Comment:
This makes me wonder on a philosophy thing for our login screen. Should we present all options, even ones not valid on the platform?

Reply:
At one time we were thinking about having them listed but disabled / grey'd out.

Reply:
Currently, all options will be shown in UI on every platform.
But there will be a message implying "Unsupported" if the platform isn't right